.vib/celerybeat-schedule.bak

```diff
@@ -0,0 +1,4 @@
+'entries', (2048, 531)
```

Comment:
What's this file? Seems to be something temporary

Reply:
➕ blocking PR until there's a response about this